.github/workflows/check-stale-keys.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           cd ssh
           git ls-tree -r --name-only HEAD | while read filename; do
            [[ "$(git log -1 --format="%at" -- $filename)" < "$(date  --date='2 years ago' +%s)"  ]] && echo "$(git log -1 --format="%ad" --date=format:'%Y-%m-%dT%H:%M:%S%z' -- $filename) $filename"
-          done || : >> old_keys.txt
+          done >> old_keys.txt || :
```

Comment:
for readability, could also use `/bin/true` to force the return code

Reply:
That is much nicer!